inst/pages/10_manipulation.qmd

```diff
@@ -181,7 +192,11 @@ When total abundances of certain phyla are of relevance, the data is initially a
 tse_phylum <- tse %>% mergeFeaturesByRank(rank = "Phylum")
 
 # Subset by feature and remove NAs
-tse_phylum_subset_by_feature <- tse_phylum[rowData(tse_phylum)$Phylum %in% c("Actinobacteria", "Chlamydiae") & !is.na(rowData(tse_phylum)$Phylum), ]
+tse_phylum_subset_by_feature <- tse_phylum[
+                                rowData(tse_phylum)$Phylum %in% 
+                                c("Actinobacteria", "Chlamydiae")
+                                &!is.na(rowData(tse_phylum)$Phylum),
```

Comment:
is this line needed btw (can it be NA if the first condition is met?)

Reply:
It was not needed indeed, removed the na check